code/modules/mob/living/carbon/human/species_types/jellypeople.dm

```diff
@@ -14,13 +14,13 @@
 /datum/species/jelly/on_species_loss(mob/living/carbon/C)
 	if(regenerate_limbs)
 		regenerate_limbs.Remove(C)
-	C.remove_language(/datum/language/slime)
+	C.remove_language(/datum/language/slime, TRUE)
 	C.faction -= "slime"
 	..()
 
 /datum/species/jelly/on_species_gain(mob/living/carbon/C, datum/species/old_species)
 	..()
-	C.grant_language(/datum/language/slime)
+	C.grant_language(/datum/language/slime, TRUE)

```

Comment:
No, you're still using it wrong.

Shadow languages only apply if it's on a body's language_datum that's being occupied by a mind.